libs/serialization/include/hpx/serialization/input_archive.hpp

```diff
@@ -180,41 +177,6 @@ namespace hpx { namespace serialization {
             load_integral_impl(ul);
             val = static_cast<T>(ul);
         }
-
-#if defined(BOOST_HAS_INT128) && !defined(__NVCC__) && !defined(__CUDACC__)
-        void load_integral(boost::int128_type& t, std::false_type)
-        {
-            load_integral_impl(t);
-        }
-
-        void load_integral(boost::uint128_type& t, std::true_type)
-        {
-            load_integral_impl(t);
-        }
-
-        // On some platforms (gcc) std::is_integral<int128>::value
-        // evaluates to false. Thus, these functions re-route the
-        // serialization for those types to the proper implementation
-        void load_bitwise(boost::int128_type& t, std::false_type)
-        {
-            load_integral_impl(t);
-        }
-
-        void load_bitwise(boost::int128_type& t, std::true_type)
-        {
-            load_integral_impl(t);
-        }
-
-        void load_bitwise(boost::uint128_type& t, std::false_type)
-        {
-            load_integral_impl(t);
-        }
-
-        void load_bitwise(boost::uint128_type& t, std::true_type)
-        {
-            load_integral_impl(t);
-        }
-#endif
```

Comment:
Not sure about removing the 128 bit integral types here. OTOH, we might have a replacement already, actually: `util::uint128_t`/`util::int128_t`?

Reply:
As far as I understand those are indeed replacements. I couldn't find any direct uses of Boost's 128 bit types, except for the overloads.

Reply:
It's probably safe to remove those. If somebody needs support for serializing 128bit integral types it can be added easily.